modules/ROOT/edits/getting-started-service-mesh.adoc

```diff
@@ -0,0 +1,17 @@
+= Get Started with Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, cluster, get started
+
+image::get-started-with-service-mesh-breadcrumb.png[75%, 75%]
+
+Use the information in this section to understand how to prepare, install, and use Anypoint Service Mesh.
+
+*Step 1:* Review Prerequisites--Before you begin, you must carefully review the requirements listed in this section. The prerequisites include, software applications, roles and permissions, and licensing requirements. Omitting to fulfill any one of these requirements might result in an incomplete or failed installation.
```

Comment:
```suggestion
*Step 1:* Review Prerequisites
Before you begin, you must carefully review the prerequisites for software applications, roles and permissions, and licensing. Otherwise, your installation will be incomplete or fail altogether.
```

Reply:
Done! Thanks!